pkgs/development/libraries/gstreamer/good/default.nix

```diff
@@ -102,8 +111,9 @@ stdenv.mkDerivation rec {
     gdk-pixbuf
     aalib
     libcaca
-    libsoup
+    libsoup_3
     libshout
+    libxml2
```

Comment:
Why is this added?  I fail to see any reason or comment about it.

Reply:
libsoup_3 does not propagate it.